gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -940,12 +949,12 @@ MM_VerboseHandlerOutput::handleConcurrentEnd(J9HookInterface** hook, UDATA event
 	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
 	UDATA contextId = stats->_cycleID;
 	char tagTemplate[200];
-	getTagTemplate(tagTemplate, sizeof(tagTemplate), _manager->getIdAndIncrement(), getConcurrentTypeString(), contextId, omrtime_current_time_millis());
+	const char* reasonForTermination = getConcurrentTerminationReason(hook, eventNum, eventData);
```

Comment:
We don't need to pass in all (hook, eventNum, eventData) to getConcurrentTerminationReason, we can simply pass in the stats struture. Or anther possibility is that getConurrentTerminationReason can be moved to the collector.

But let's just leave it since handleConcurrentEndInternal did it like this and you're mimicking that

Reply:
After seeing J9 changes, it's more appropriate to change make the change